source/features/repo-age.tsx

```diff
@@ -15,15 +15,21 @@ const dateFormatter = new Intl.DateTimeFormat('en-US', {
 });
 
 const getRepoCreationDate = cache.function(async (): Promise<string> => {
-	const {repository} = await api.v4(`
-		repository(${getRepoGQL()}) {
-			createdAt
-		}
-	`);
+	const commitsCount = Number(select('li.commits .num')!.textContent!.trim()!.replace(',', ''));
+	const lastCommitHash = select('.commit-tease .commit-tease-sha')!.getAttribute('href')!.split('/').pop();
 
-	return repository.createdAt;
+	if (!commitsCount || !lastCommitHash) {
+		return Promise.reject(new Error('Cannot get commit count or last commit hash'));
+	}
```

Comment:
If these values aren't available, those `select` will fail. 

Also since this is an async function you can just throw an error and it becomes a rejected promise.

Reply:
For 2nd point, I tried just throwing an error but ts compiler was complaining 🤔 but now I tried again and it isn't complaining! 🤦‍♂ Not throwing the error now just returning if needed information is not available.

Reply:
The only check here should be `if count === 0, return false`

I think `undefined` won’t be cached and we always expect the hash to be there. If it’s not, it’s an error and it should appear in the console automatically.